### PR TITLE
Add worker dashboard

### PR DESCRIPTION
**Pull request by TomAugspurger**
xref https://github.com/dask/dask-gateway/issues/280#issuecomment-650362775

I'm unsure about tests for this...

### CODE REVIEW

File: dask-gateway/dask_gateway/dask_cli.py [hunk 1]
```diff
@@ -576,6 +583,8 @@ async def start_worker(
     scheduler_address=None,
     local_directory="",
     nanny=True,
+    dashboard=True,
```
Comment (TomAugspurger) [inline dask-gateway/dask_gateway/dask_cli.py:586]: The default of `True` here matches the `dask-worker` CLI default. It differs from the `distributed.Worker` default of False. I think following the CLI default is appropriate.

**Comment by TomAugspurger** [comment]
I'm not sure why, but my latest commits aren't showing up here.

**Comment by jcrist** [comment]
github is having issues right now :/

**Comment by TomAugspurger** [comment]
@jcrist all green. Did you have any thoughts on the need for tests here?

**Comment by jcrist** [comment]
Fine by me. I'm hoping to remove the need for the `dask-gateway-worker` cli soon™, don't think we need to add tests for kwarg forwarding that should be compatible with the original cli. Thanks for working on this!

**Pull request merged**